src/parser.js

```diff
@@ -1194,11 +1194,13 @@ export default class Parser {
         element.attributes.set('samples', e);
       }
     } else {
-      element = new Type();
+      let example;
 
       if (parameter['x-example'] !== undefined) {
-        element.content = parameter['x-example'];
```

Comment:
Note, if we set the `.content` of an element, when the new value is an array, the array has to be mimim elements. Whereas when we pass the value to the constructor of an Array element it will refract all of the given items in the array.